src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -74,18 +83,18 @@ class boss_gluth : public CreatureScript
             me->SetReactState(REACT_AGGRESSIVE);
         }
 
-        void MoveInLineOfSight(Unit *who)
+        void MoveInLineOfSight(Unit *who) override
         {
             if ((!me->GetVictim() || me->GetVictim()->GetEntry() != NPC_ZOMBIE_CHOW) && who->GetEntry() == NPC_ZOMBIE_CHOW && me->IsWithinDistInMap(who, 6.5f))
```

Comment:
Maybe breaking this into 2 Ifs eases readability, what do you think?

Reply:
Sure